gadget/gadget_test.go

```diff
@@ -3041,6 +3041,108 @@ func (s *gadgetYamlTestSuite) TestLayoutCompatibilityWithImplicitSystemData(c *C
 	c.Assert(err, IsNil)
 }
 
+var mockEncDeviceLayout = gadget.OnDiskVolume{
+	Structure: []gadget.OnDiskStructure{
+		// Note that the first ondisk structure we have is BIOS Boot, even
+		// though "in reality" the first ondisk structure is MBR, but the MBR
```

Comment:
http://www.unnecessaryquotes.com/ :-)

Reply:
yOu CaN tAkE mY uNeCeSsArY qUoTeS fRoM mY uNeCeSsArIlY "qUoTeD hAnDs"